docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,575 @@
+- [Example configurations](#example-configurations)
+
+## Feature overview
+
+In the V1-LTS version of the CLI, users issue commands from the `zowe profiles` group to create, edit, and manage user profiles. Each profile contains the host, port, username, and password for a specific mainframe service instance. While that approach is effective, users often need to duplicate values across profiles and spend time managing many profiles separately.
+
+The **global profile functionality** simplifies profile management by letting you edit, store, and share mainframe configuration details in one location. You can use a text editor to populate configuration files with connection details for your mainframe services.
+
+### Benefits
+
+Global profile configuration can improve your Zowe CLI experience in the following ways:
+
+- As a CLI user, you can manage your connection details efficiently in one location.
+- As a team leader, you can share a configuration file with your team members so that they can easily access mainframe services. You can add the file directly to your project in an SCM.
+- As a new team member, you can onboard quickly by consuming your team's configuration file.
+
+### Changes to secure credential storage
+
+In this version, Secure Credential Store (SCS) Plug-in is deprecated. The `zowe scs` and `zowe config` command groups are obsolete. The equivalent functionality that encrypts your credentials is now included in the core CLI.
+
+With the new configuration, the CLI prompts you to enter username and password securely by default. Commands in the new `zowe config` command group let you manage security for any option value.
+
+## Installing @next version
+
+To get started, install the Zowe CLI `@next` version from the online registry. You can follow this procedure for a first-time installation, or to update a currently installed version.
+
+**Follow these steps:**
+
+1. Meet the [software requirements for Zowe CLI](https://docs.zowe.org/stable/user-guide/systemrequirements.html#zowe-cli-requirements).
+
+2. To install or update the core CLI, open a command-line window and issue the following command:
+
+   ```
+   npm install -g @zowe/cli@next
+   ```
+
+3. Meet the [software requirements for each plug-in](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html#software-requirements-for-zowe-cli-plug-ins).
+
+4. (Optional) Check [npmjs.com](https://www.npmjs.com/) for any Zowe plug-ins that have an `@next` version available. If an `@next` version is available, you can issue the following command (substituting the plugin name) to install it. If no `@next` version, try installing the `@latest` version of the plug-in:
+
+    ```
+    zowe plugins install @zowe/<plugin-name>@next
+    ```
+
+   Zowe CLI and optional plug-ins are installed!
+
+5. If you previously had an instance of Zowe CLI installed, your old configuration files are no longer used in this version. Delete the following files from your local `.zowe/` directory:
```

Comment:
These files are respected in the new version. However, they are ignored if zowe.config.json is found either globally or at the project level (or up the directory structure).

Reply:
Not sure if I picked up what you laid down. How does this sound?

"If you previously had an instance of Zowe CLI installed, your current configuration files are ignored if zowe.config.json is found either globally or at the project level (or up the directory structure)."